activation/activation.go

```diff
@@ -3,6 +3,7 @@
 package activation
 
 import (
+	"context"
```

Comment:
Thanks for noticing. Actually, I didn't add the context imports to the GRPC code in this PR. It was there from before. The google GRPC stack may be doing something different with context so I think this can be left alone.

Reply:
okay, will fix this, thanks guys